src/nvim/mpack/LICENSE-MIT

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2016 Thiago de Arruda
```

Comment:
should this go in `src/mpack` ? In https://github.com/neovim/neovim/issues/11239 we plan to move xdiff to `src/xdiff/` . Rationale: 

1. signals that we don't "own" this library
2. thereby leaves the door open for un-vendoring it in the future?

Reply:
I don't have a strong opinion either way, but for the records, who "owns" the code? unlike xdiff, there's no upstream development to speak of, so _if_ the need to change stuff arises, I plan to change it directly in the code here. ( like lmpack.c might need a C API so we can call it directly when we have parsed `[REPLY, {id-from-lua-vim.rpcrequest-call}, ` in C and unpack the return value directly to lua without any C allocations). This doesn't preclude upstreaming the changes back and un-vendoring later, of course.

Reply:
> This doesn't preclude upstreaming the changes back and un-vendoring later, of course.

yeah, it's mostly just a signal for us/contributors. Things in `src/*` aren't necessarily frozen, but if there are no downsides it probably makes sense to put things there if we have only "weak" (or none) ownership of them?

Reply:
done.